### ISSUE TITLE
histogram_quantile results sometimes doesn't work.

### ISSUE THREAD

**Issue by ijij0516** [VictoriaMetrics/VictoriaMetrics#154]
![スクリーンショット 2019-08-20 15 52 21](https://user-images.githubusercontent.com/5616330/63325339-de958e80-c364-11e9-8235-00f798400861.png)
![スクリーンショット 2019-08-20 15 52 13](https://user-images.githubusercontent.com/5616330/63325350-e5240600-c364-11e9-889b-9b73b008482a.png)

First is Prometheus result, and second one is VictoriaMetrics.
#136 seems to be a similar issue.
Do you have any ideas?
The query is as follows.
```
histogram_quantile(0.5, sum(irate(nginx_http_request_duration_seconds_bucket{kubernetes_namespace=~"prd",host!="", kubernetes_pod_name=~".*prd-app-59748c9bdc-25zq5"}[2m])) by (app,le))
```

Thank you.

**Comment by valyala** [comment]
1. Which version of VictoriaMetrics do you use? It can be located at `/metrics` page via `vm_app_version` metric or it can be obtained via passing `-version` argument to VictoriaMetrics binary?

2. Could you attach Prometheus and VictoriaMetrics graphs for `sum(irate(nginx_http_request_duration_seconds_bucket{kubernetes_namespace=~"prd",host!="", kubernetes_pod_name=~".*prd-app-59748c9bdc-25zq5"}[2m])) by (app,le)` on the same time range?

3. Try substituting `irate` with `rate` in the query, since `irate` doesn't play - https://medium.com/@valyala/why-irate-from-prometheus-doesnt-capture-spikes-45f9896d7832

**Comment by ijij0516** [comment]
Thank you for the reply!

1. The VictoriaMetrics version is `v1.25.2-cluster` and it is running on Kubernetes.  
Prometheus version is v2.11.0.
![スクリーンショット 2019-08-22 10 04 34](https://user-images.githubusercontent.com/5616330/63478401-a7d68a00-c4c4-11e9-998a-f50e0dca5971.png)

2.  First Prometheus, second VictoriaMetrics.<img width="1280" alt="スクリーンショット 2019-08-22 10 08 43" src="https://user-images.githubusercontent.com/5616330/63478487-f421ca00-c4c4-11e9-90d8-b1f99a0c0c1a.png"><img width="1280" alt="スクリーンショット 2019-08-22 10 08 54" src="https://user-images.githubusercontent.com/5616330/63478506-fedc5f00-c4c4-11e9-8dcc-bb162f1aa992.png">

3. I substituted irate with rate, but still some data are dropped from the graph.<img width="1276" alt="スクリーンショット 2019-08-22 10 21 32" src="https://user-images.githubusercontent.com/5616330/63478906-b9209600-c4c6-11e9-9b33-b9ed08581c87.png"><img width="1280" alt="スクリーンショット 2019-08-22 10 21 43" src="https://user-images.githubusercontent.com/5616330/63479033-4fed5280-c4c7-11e9-98bd-db3b795ef357.png">
First Prometheus, second VictoriaMetrics.
The query is as follows.

```
histogram_quantile(0.5, histogram_quantile(0.5, sum(rate(nginx_http_request_duration_seconds_bucket{kubernetes_namespace=~"prd",host!="", kubernetes_pod_name=~".*prd-app-59748c9bdc-25zq5"}[2m])) by (app,le))
```

**Comment by valyala** [comment]
@ijij0516 , could you try building VictoriaMetrics from 1272e407b2f62dc4e7e15e3644b76c552eb6b16a (single-node version) or e9db22a551268cfbb2836946f7b756e66c35495e (cluster version) and verifying whether the issue is fixed?

**Comment by valyala** [comment]
@ijij0516 , the bugfix is available in [v1.26.0](https://github.com/VictoriaMetrics/VictoriaMetrics/releases). Could you verify it there?

**Comment by ijij0516** [comment]
@valyala I confirmed that the problem has been solved in v1.26.0. now. 
Thank you!
Prometheus
<img width="1271" alt="スクリーンショット 2019-08-23 10 19 19" src="https://user-images.githubusercontent.com/5616330/63560249-53004580-c590-11e9-9f8c-5320c849e35c.png">
VictoriaMetrics
<img width="1276" alt="スクリーンショット 2019-08-23 10 19 27" src="https://user-images.githubusercontent.com/5616330/63560277-61e6f800-c590-11e9-8300-3f0324d0dc8f.png">
